### DISCUSSION TITLE
Learn disappeared

### DISCUSSION THREAD

**Discussion by KLelong** [rspamd/rspamd#4231] [Q&A]
Hi,
I'm using rspamd since a few weeks now on my home-server.
I was feeding the bayes spam-filter via the web-interface with success.
When today I tried to "teach" a new spam message I could not. The menu item "scan/learn" is replaced by "scan". What happened ?
I did not modify any rspamd-configuration. Is there anything in the logs I should be looking for ?
I already tried to restart rspamd with no success.

I found another thread (#4188) which resembles my problem, so I applied the answer to my apache server, no luck, so I reverted that.
I traced the rspamd log when logging in. When accessing the start page, I get a request for the password in the browser and the log produces this : 
`2022-07-27 18:45:04 #1113808(controller) <4a4686>; csession; rspamd_controller_check_password: absent password has been specified; source ip: 127.0.0.1:40464
`
Logging in produces no log-entry
 
local.d/worker-controller.inc contains this : 
```
password = "<obfuscated>"
secure_ip = "127.0.0.1";
secure_ip = "192.168.185.1/24";
secure_ip = "::1";
```
Any thoughts ?

TIA,
Koenraad.

**Comment by moisseev** [comment]
It may be due to #4219.
